src/cls/IPM/ResourceProcessor/Default/Document.cls

```diff
@@ -172,6 +172,26 @@ Method OnPhase(pPhase As %String, ByRef pParams, Output pResourceHandled As %Boo
                 Set tSC = $$$OK
                 Quit
               }
+              If ..ResourceReference.Deploy {
+                If tVerbose {
+                  Write !, "Skipping deployed resource: " _ ..ResourceReference.Name
+                }
+                Set ..CompileFromProject = 1
+                Set tDeployedDirectory = ##class(%File).NormalizeDirectory("studio-project", ..ResourceReference.Module.Root)
+                $$$ThrowOnError($System.OBJ.Load(tDeployedDirectory_"Deployed.xml", "<none>", , .tLoadedList))
```

Comment:
Should Load() be used to import studio projects? Usually we use %Studio.Project:ImportFromFile(). This will also ensure any ^INSTALL routine packaged with the studio project is run. Not sure how we want to handle the ^INSTALL routine in this case though (or if we even want it to run) since this isn't really the same concept as packaging the entire module as a studio project so it can be imported in a context without IPM. Here, we would import the studio project in the reload phase and then continue with regular activate/configure to run invokes and installer class code so don't need the ^INSTALL routine. @isc-tleavitt thoughts?

Reply:
I don't think we should support use of ^INSTALL in an IPM context. Better to use invokes to be more explicit/transparent about what's happening.

Reply:
In that case, should ensure ^INSTALL isn't packaged in the studio project in this case. I know there is some logic in the studio project lifecycle class for it which is why I asked.